demo/chat/src/containers/messages/Messages.jsx

```diff
@@ -257,21 +263,21 @@ const Messages = ({
             <ChatBubble
               variant={variant}
               senderName={m.senderName}
-              content={m.content}
               redacted={m.redacted}
               showName={showName}
               showTail={showTail}
             >
-              {m.editedNote}
-              {m.metadata ? (
+              <div>
+                {m.content}
+                {m.editedNote}
```

Comment:
What happens when you have `content` and an `editedNote`? Not exactly sure what `editedNote` refers to, but is it intentional to just display all of it as a single group of text?

Reply:
`editedNote` refers to a string that gets added to a chat message like *edited yesterday at 1:00pm*. I made sure this is still formatted properly in the demo.